### ISSUE TITLE
Different widget form factor (2w*4h)

### ISSUE THREAD

**Issue by Der-Flip** [thundernest/k-9#6352]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.202

### Problem you are trying to solve

Well, put simply: The widget size of 4 wide by 3 high is too large. 

### Suggested solution

Coming from blackberry I tried the blackberry client for Android only to be disappointed and after about a year I've had it now.
What I do enjoy is the widget though. The size of 2 wide by 4 tall is perfect to allow a rough but good overview of the mail inboxes while leaving enough space for further apps.
If k-9 would support a 2*4 widget I'd switch for good tbh.

### Screenshots / Drawings / Technical details

Ignoring the wallpaper you can see 2*4 allows for essential apps to be easy to reach on the first page allowing it to be the perfect hub for everything necessary
Using the current 4*3 sized widget there'd be a lot space wasted and would make this efficient layout impossible.

<img src="https://user-images.githubusercontent.com/18213041/194208817-7a2cdcc6-9cb0-4285-bce8-63b35162da05.jpg" width="33%">

edit: reduced image display size

**Comment by cketti** [comment]
K-9 Mail's message list widget can be resized. On most Android versions this works by long-pressing the widget on your home screen.

**Comment by Der-Flip** [comment]
ahh, thanks, I'm still fairly new to android and learning the ins and outs. I tried that but must have overlooked that faint blue outline that I noticed, now.

sadly the width is only reducible to 3 wide, is there a chance to have this limit reduced to 2?
I do admit that 2 is very narrow. I don't need the widget to do much work though, I mostly need to know which inbox received messages from whom, so I know whether I need to get active or not.
